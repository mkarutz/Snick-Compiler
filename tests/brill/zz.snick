proc f (ref float i)

    i := i / (0 + i) * 8;
end

proc main ()
    int x;
    int y;
    float m;
    float n;

    x := (3 + 3 + 8) / 4;
    y := -5 + x;
    m := y;
    while x > 1 do
        if m > 0 then
            write m;
            write n;
            if x / 0 < m then
                f(n);
            fi
        else
            f(m);
        fi
    od
end
