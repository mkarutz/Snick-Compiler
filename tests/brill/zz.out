8.0000000.000000DIVIDE BY ZERO
